### PR TITLE
update code examples adding the new `--allow-relay-address` argument to `project ticket`

### PR DESCRIPTION
**Pull request by adrianbenavides**
Breaking changes adaptations:
- Add `--relay` argument to `project ticket` command
- Replace the word `token` by `ticket` when referring to the result of running the `project ticket` command
- Remove the use of `project enroll --project-path`, or replace it with `project import --project-file`

Simplifications:
- Simplify `--to` argument for `tcp-inlet create` to reduce boilerplate using its default value

### CODE REVIEW

File: reference/command/secure-channels.md [hunk 1]
```diff
@@ -148,10 +148,10 @@ Let's create an end-to-end secure channel through an elastic relay in your Orche
 The [<mark style="color:blue;">Project</mark>](nodes.md#project) that was created when you ran `ockam enroll` offers an Elastic Relay Service. [<mark style="color:blue;">Delete any existing nodes</mark>](nodes.md#nodes) and then try this new example:
 
 ```
-» ockam project information --output json > project.json
+» ockam enroll
 
-» ockam node create a --project-path project.json
-» ockam node create b --project-path project.json
```
Comment (adrianbenavides) [inline reference/command/secure-channels.md:154]: The `--project-path` argument doesn't exist anymore and we now have the `ockam project import` command. But why do we want to add this extra step if this example is not a multi-machine example?

File: reference/command/credentials.md [hunk 2]
```diff
@@ -132,10 +132,8 @@ HELLO
 » ockam reset -y
 » ockam enroll
 
-» ockam project information --output json > project.json
-
-» ockam node create a --project-path project.json
-» ockam node create b --project-path project.json
```
Comment (adrianbenavides) [inline reference/command/credentials.md:138]: The --project-path argument doesn't exist anymore and we now have the ockam project import command. But why do we want to add this extra step if this example is not a multi-machine example?

Comment (etorreborre) [reply reference/command/credentials.md:138]: Yes that seems unnecessary in that case.

File: reference/command/secure-channels.md (hunk 1 above)
Comment (etorreborre) [reply reference/command/secure-channels.md:154]: Yes that seems unnecessary in that case.

**Review by etorreborre — APPROVED** [review]
🧪 that looks good to me

**Pull request merged**